vcd/writer.py

```diff
@@ -403,6 +404,8 @@ def _finalize_registration(self):
         assert self._registering
         print(*self._gen_header(), sep='\n', file=self._ofile)
         if self._ident_values:
+            print('#' + str(int(self._init_timestamp)), file=self._ofile)
+            self._prev_timestamp = self._init_timestamp
```

Comment:
With my above suggestion to eliminate `_init_timestamp`, we can use `_prev_timestamp` for the timestamp here and eliminate this assignment.